src/external/rawspeed/rawspeed-identify.cpp

```diff
@@ -119,7 +119,11 @@ int main(int argc, const char* argv[])
 
     fprintf(stderr, "Loading file: \"%s\"\n", argv[1]);
 
+    #if defined (_WIN32)
```

Comment:
@houz I'm sure `WIN32` is fine in MinGW and will always be for compatibility reasons. However, the predefined macros should start with an underscore, as those names are implementation-reserved. `WIN32` could be redefined by the user, but it's illegal to do that for `_WIN32`.

So they both work, but one is wrong in theory.